### DISCUSSION TITLE
How to build this github project

### DISCUSSION THREAD

**Discussion by Lorenzo-apd** [community/community#29272] [New to GitHub]
Hello,
I wanted to play around with [this](https://github.com/ange-yaghi/engine-sim) project and try to learn something from it, but I can't figure our how to run it.
I installed CMake and cloned the project, now I am stuck and I don't know how to install dependencies, make sure they are found by cmake and build and run the project.

**Comment by ghost** [comment]
Did you try downloading the file. You can also download with github desktop if you have that installed.

**Reply by Lorenzo-apd** [reply]
I have downloaded the project, I don't know how to install dependencies and get the project running.

**Comment by airtower-luna** [comment]
Looks like you're not the only one with that question, you might want to follow this issue in the linked repository: https://github.com/ange-yaghi/engine-sim/issues/46

Obviously I can't promise if there'll be any response, but at least an issue in the repository is much more likely to be seen by the repository owner.

**Reply by Lorenzo-apd** [reply]
Thank you, will check the issue and see if any further solutions are given.
